packages/foam-vscode/src/features/dataviz.ts

```diff
@@ -23,6 +33,7 @@ const feature: FoamFeature = {
         noteAddedListener.dispose();
         noteUpdatedListener.dispose();
         noteDeletedListener.dispose();
+        panel = undefined;
```

Comment:
```suggestion
        panel.dispose();
```

Reply:
This is per-VSCode documentation https://code.visualstudio.com/api/extension-guides/webview